src/stores/Nav2.ts

```diff
@@ -21,7 +21,7 @@ import PageProfileUpdate from 'pages/PageProfileUpdate'
 import PageSettingsDebug from 'pages/PageSettingDebug/PageSettingsDebug'
 import PageSettingsOther from 'pages/PageSettingsOther'
 import PageSettingsProfile from 'pages/PageSettingsProfile'
-import PageTransferDial from 'pages/PageTransferDial'
+import PageTransferDial from 'pages/PageTransferDial/PageTransferDial'
```

Comment:
create index file for this. so your import will be shorten like `import PageTransferDial from 'pages/PageTransferDial`